unit-tests/func/frames/test_pipeline_first_frame_delay.py

```diff
@@ -0,0 +1,61 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+# test:device L500*
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test
+import time
+import platform
+
+
+# Start depth + color streams and measure the time from stream opened until first frame arrived using pipeline API.
+# Verify that the time do not exceeds the maximum time allowed
+# Note - Using Windows Media Foundation to handle power management between USB actions take time (~27 ms)
+
+
+def time_to_first_frame(config):
+    pipe = rs.pipeline()
+    start_call_stopwatch = Stopwatch()
+    pipe.start(config)
+    pipe.wait_for_frames()
+    delay = start_call_stopwatch.get_elapsed()
+    pipe.stop()
+    return delay
+
+
+time.sleep(3)  # The device starts at D0 (Operational) state, allow time for it to get into idle state
+
+max_delay_for_depth_frame = 5
+max_delay_for_color_frame = 5
+
+# Set maximum delay for first frame according to product line
+dev = test.find_first_device_or_exit()
+
+product_line = "Unknown"
+if dev.supports(rs.camera_info.product_line):
```

Comment:
I think we can pretty much require that the device supports a product line, no? I think you can just skip this test...

Reply:
Changed